Restructure docs into sub-folder and enhance README to signpost the demo (#53)
CCC-Attestation/attested-tls-poc: README.md

either the client, the server, or both can attest themselves, leveraging any hardware backend, and
using different verification topologies.

> To showcase the standardization effort, we are also developing this open-source and end-to-end

Reviewer: ```suggestion
To showcase the standardization effort, we are also developing an open-source end-to-end
```
